<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	border-image: url(:/backgroud/image/d2768e4ea47d92877faddcbf67e94367.jpeg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-30</y>
      <width>1081</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/1.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/backgroud/image/15.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_back">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>570</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(255,0, 255);</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_lightup">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255, 127);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>亮度增加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_guassion">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>230</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>高斯</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_flipupdown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>上下翻转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_lightdown">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>50</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>亮度减少</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_flipleftright">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>左右翻转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_cat">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>拼接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_cut">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>裁剪大小</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>旋转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_cold">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>冷色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_warm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string>暖色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_big">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>130</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>放大</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>280</y>
       <width>351</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/backgroud/image/15.png);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_media">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>320</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>中值滤波</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="file">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
   </widget>
   <addaction name="file"/>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>input</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
